server/wfnews-persistence/src/main/resources/ca/bc/gov/nrs/wfnews/persistence/v1/dao/mybatis/mapper/PublishedIncidentMapper.xml

```diff
@@ -299,6 +302,31 @@
 		  FROM WFNEWS.PUBLISHED_INCIDENT_DETAIL
 		  WHERE PUBLISHED_INCIDENT_DETAIL_GUID IS NOT NULL
 		  AND PUBLISHED_INCIDENT_DETAIL_GUID != ''
+		  <if test="fireOfNote != null and fireOfNote == true">
+				AND fire_of_note_ind = 'Y'
+		  </if>
+		  <if test="fireOfNote != null and fireOfNote == false">
+				AND fire_of_note_ind = 'N'
+		  </if>
+		  <if test="out == true">  
+				AND stage_of_control_code = 'OUT'
+		  </if>
+		  <if test="out == false">  
+				AND stage_of_control_code != 'OUT'
+		  </if>
+		  <if test="fireCentre != null &amp;&amp; fireCentre != ''">  
+				AND fire_centre = #{fireCentre, javaType=java.lang.String, jdbcType=VARCHAR, mode=IN}
+		  </if>
+		  <if test="xmin != null &amp;&amp; ymin != null &amp;&amp; xmax != null &amp;&amp; ymax != null">
+				AND geometry &amp;&amp; ST_MakeEnvelope (#{xmin, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, #{ymin, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, #{xmax, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, #{ymax, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, 4326)
+		  </if>
+		  <if test="latitude != null &amp;&amp; longitude != null &amp;&amp; radius != null">
+				AND ST_DWithin(geometry, ST_SetSRID(ST_MakePoint(#{longitude, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, #{latitude, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}), 4326), #{radius, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN})
```

Comment:
Looks like this merge will blow away changes made for the radius check to use metres instead of degress for the distance calculation.

Can you update this in your branch to use:
```
AND ST_DWithin(ST_Transform(geometry, 3857), ST_Transform(ST_SetSRID(ST_MakePoint(#{longitude, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}, #{latitude, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN}), 4326), 3857), #{radius, javaType=java.lang.Double, jdbcType=DOUBLE, mode=IN})
```

If not, we'll have to remember to reapply. Not sure why github is determining this to be a clean replace and not a merge conflict considering the PR came in just before this one, but whatever.

Reply:
Actually nvm, I can merge and do it now